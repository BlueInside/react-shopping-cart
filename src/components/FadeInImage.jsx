function FadeInImage() {
  return <div></div>;
}

export default FadeInImage;
