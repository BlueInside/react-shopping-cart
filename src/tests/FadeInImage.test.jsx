import { render } from '@testing-library/react';
import { describe, expect, it } from 'vitest';
import FadeInImage from '../components/FadeInImage';

describe('FadeInImage component', () => {
  it('render FadeInImage correctly', () => {
    const { container } = render(<FadeInImage />);
    expect(container).toMatchSnapshot();
  });
});
