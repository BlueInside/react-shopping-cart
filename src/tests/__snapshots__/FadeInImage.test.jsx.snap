// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`FadeInImage component > render FadeInImage correctly 1`] = `
<div>
  <div />
</div>
`;
